src/back/models/Project.ts

```diff
@@ -29,14 +35,39 @@ export default class ProjectModel extends Model<ProjectAttributes> {
 
   static async getProject(id: string) {
     const query = SQL`
-      SELECT pr.*, p.user as author, array_agg(co.address) as coauthors
-      FROM projects pr
-      JOIN proposals p on pr.proposal_id = p.id
-      LEFT JOIN coauthors co on pr.proposal_id = co.proposal_id AND co.status = 'APPROVED'
-      WHERE pr.id = ${id}
-      GROUP BY pr.id, p.user;
+        SELECT
+            pr.*,
+            p.user as author,
+            COALESCE(json_agg(
+                     json_build_object(
+                             'id', pe.id,
+                             'project_id', pe.project_id,
+                             'address', pe.address,
+                             'name', pe.name,
+                             'role', pe.role,
+                             'about', pe.about,
+                             'relevantLink', pe."relevantLink",
+                             'status', pe.status,
+                             'updated_by', pe.updated_by,
+                             'updated_at', pe.updated_at,
+                             'created_at', pe.created_at
+                     ) ORDER BY pe.id) FILTER (WHERE pe.id IS NOT NULL), '[]') AS personnel,
+            COALESCE(array_agg(co.address) FILTER (WHERE co.address IS NOT NULL), '{}') AS coauthors
+        FROM ${table(ProjectModel)} pr
+                 JOIN ${table(ProposalModel)} p ON pr.proposal_id = p.id
+                 LEFT JOIN ${table(PersonnelModel)} pe ON pr.id = pe.project_id AND pe.deleted = false
+                 LEFT JOIN ${table(CoauthorModel)} co ON pr.proposal_id = co.proposal_id AND co.status = ${
+      CoauthorStatus.APPROVED
+    }
+        WHERE pr.id = ${id}
```

Comment:
Are we validating this is an id?

Reply:
We are in the route handler, but I'll add a check in the model for xtra safu